_non-cosmos/ethereum/assetlist.json

```diff
@@ -1127,6 +1127,36 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.svg"
         }
       ]
+    },
+    {
+      "description": "Gelotto blends gaming, DEFI, and fun Dapps in a unique Web3 ecosystem. Join us to become part of the house, earning from all Gelotto products. Our main goal? To bring the masses into Web3 with open, accessible, and enjoyable Dapps.",
+      "type_asset": "erc20",
+      "address": "0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2",
+      "denom_units": [
+        {
+          "denom": "0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "glto",
+          "exponent": 6
+        }
+      ],
+      "base": "0xd73175f9eb15eee81745d367ae59309Ca2ceb5e2",
+      "name": "Gelotto",
+      "display": "glto",
+      "symbol": "GLTO",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/glto.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/glto.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/glto.png",
```

Comment:
Please update URL to point to the /juno/images/ directory.
Also applies to logo_URIs